hw/ip/alert_handler/data/alert_handler.hjson

```diff
@@ -317,17 +399,34 @@
              // TODO: remove the exclusion after set up top-level esc_receiver_driver
              "excl:CsrNonInitTests:CsrExclCheck"],
       fields: [
-        { bits: "0", name: "LA", desc: "Cause bit " }
+        { bits: "0", name: "LA", desc: "Cause bit ", resval: 0}
       ]
       }
     },
 # classes
 
+    { name:     "CLASSA_REGWEN",
```

Comment:
i'm curious, why didn't we make this CLASS0~3 and then use multireg? 
no need to change now, but just trying to remember if there was a specific reason.

Reply:
ah i forgot these are templated..so it doesn't even really matter.

Reply:
bhu honestly I think this was just the terminology we've used in the spec.